service/cloudconfigv1/cloud_config.go

```diff
@@ -57,13 +57,17 @@ func (c *CloudConfig) NewMasterTemplate(customObject kvmtpr.CustomObject, certs
 
 	// TODO remove defaulting as soon as custom objects are configured.
 	if customObject.Spec.Cluster.Version == "" {
-		customObject.Spec.Cluster.Version = string(cloudconfig.V_0_1_0)
+		customObject.Spec.Cluster.Version = string(cloudconfig.V_1_0_0)
+	}
+	// TODO remove defaulting as soon as custom objects are migrated.
+	if customObject.Spec.Cluster.Version == string(cloudconfig.V_0_1_0) {
+		customObject.Spec.Cluster.Version = string(cloudconfig.V_1_0_0)
```

Comment:
We use [`1.0.0`](https://github.com/giantswarm/k8scloudconfig/blob/ea70bc4ea6fc4c3214c0a71e3c6a95ba3be5fd32/template.go#L16-L18) now which is the same as `0.1.0` but with experimental encryption disabled. 

The weird conditionals here have to stay a bit until we reconcile on the cluster-service/kubernetesd side. There is https://github.com/giantswarm/giantswarm/issues/2148 to track this.

Reply:
Ack. We'll need to do the same in `aws-operator`.

Reply:
To be on the same page, in AWS you do not need to do anything now. What we need to make sure is that we use `2.0.0` when migrating to CRDs.

Reply:
OK, that makes sense. Thanks for clarifying.